js/script.js

```diff
@@ -1,20 +1,22 @@
 // Note: the code will still work without this line, but without it you
 // will see an error in the editor
-/* global EspLoader, ESP_ROM_BAUD, ESP32, ESP8266, port, reader, inputBuffer, generate */
+/* global reader */
 'use strict';
 
+import * as esptoolPackage from "https://unpkg.com/esp-web-flasher@5.1.1/dist/web/index.js?module"
+import {LittleFS, LfsConfig, LfsFile, LFS_O_WRONLY, LFS_O_CREAT, LFS_O_TRUNC} from 'https://cdn.jsdelivr.net/gh/adafruit/littlefs-pure-js@1.0.2/littlefs.js';
+
 const FIRMWARE_API = "//io.adafruit.com"
 const QUICK_START_LINK = "https://learn.adafruit.com/quickstart-adafruit-io-wippersnapper/installing-wippersnapper"
 const DO_DOWNLOAD = false
 
 const BOARD_TO_CHIP_MAP = {
-  'feather-esp8266': ESP8266,
-  'feather-esp32': ESP32,
-  'feather-esp32-v2-daily': ESP32
+  'feather-esp8266': esptoolPackage.CHIP_FAMILY_ESP8266,
+  'feather-esp32': esptoolPackage.CHIP_FAMILY_ESP32,
+  'feather-esp32-v2-daily': esptoolPackage.CHIP_FAMILY_ESP32
```

Comment:
Why don't these break without the `await`?

Reply:
It might be because they're just values and not an asynchronous function.

Reply:
It's not being invoked as a function in the latter case either... I'm guessing I'm confused about this `import ... from "url"` syntax, maybe `espToolpackage` is itself a promise... that also has values attached to it? Going to do some more reading, this is clearly working as you have it I'm just confused!

Reply:
Thanks. I'm not 100% sure of it either and your questions make me curious as well.